docker/py3-native/verify.py

```diff
@@ -1,4 +1,9 @@
+
 # make sure regex is working cause in new versions there are problems
 import regex
 pattern = regex.Regex('\\\\d\\+', flags=regex.V0)
-print('regex is good')
\ No newline at end of file
+print('regex is good')
+
+# make sure that the iproute command is installed in py3-native image
+import subprocess
```

Comment:
lets move this check to verify py

Reply:
debian does not have it by default, hence it cause us problems in the `verify.py` of python3, cause `python3-deb` uses it as well.